src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -0,0 +1,224 @@
+
+## Connect CodeStream and New Relic [#connect]
+
+Requirements for connecting CodeStream and New Relic:
+* [New Relic account](https://newrelic.com/signup) (If you don't have a New Relic account, sign up at [newrelic.com/signup](https://newrelic.com/signup). It's free, forever!)
+* [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+
+In order to get the most out of CodeStream and New Relic, connect CodeStream to New Relic via your [New Relic user key](/docs/apis/intro-apis/new-relic-api-keys/#user-api-key).
+
+Once you have your API key, in **Observability** click **Connect to New Relic One**, then paste your API key and click **Connect**.
+
+## Jump to the source of your errors [#errors]
+
+Requirements for opening stack tracer errors in your IDE:
+* [New Relic account](https://newrelic.com/signup) (If you don't have a New Relic account, sign up at [newrelic.com/signup](https://newrelic.com/signup). It's free, forever!)
+* [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+* [CodeStream and New Relic connection](#connect)
+* [Your repository's commit sha](#sha)
+* [Your repository's URL](#repo-url)
+* If you're using an APM agent, [configure your APM agent for CodeStream](#apm)
+* [Data being reported to New Relic from an app, host, or other entity ](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==)
+* [A workload for errors inbox](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/#create)
+* [An error](/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/#error-details)
+
+New Relic errors inbox is a single place to proactively detect, prioritize, and take action on all the errors before they impact customers. Now via CodeStream, you can jump from an error directly to the code causing the error in your IDE.
+
+<Callout variant="tip">
+    Limited to APM, browser, and mobile errors.
+</Callout>
+
+From **[one.newrelic.com/](https://one.newrelic.com/)**, go to **Errors Inbox**, click a stack trace error, then click **Open in IDE**.
+
+### Connect a stack trace error to the correct version of code [#sha]
+
+For the **Open in IDE** button to open the version of your code that generated the stack trace error, you'll need your [commit sha](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection)
+and/or your [release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) associated with the running version of your software.  
+
+CodeStream only needs the first seven characters of your commit sha (for example, `734713b`) to make this connection, but you can include the entire sha.  
+
+Alternately, you can use a release tag (such as `v0.1.209` or `release-209`) for CodeStream to find the correct version of your code.
+
```

Comment:
The setup is different for Browser and Mobile

For Mobile, we need to tell the customer that they need to create git tags that match their appVersion because all telemetry is already linked to appVersion.  https://docs.newrelic.com/attribute-dictionary/?event=MobileReques
<img width="1071" alt="Screen Shot 2021-10-13 at 9 23 57 PM" src="https://user-images.githubusercontent.com/47154052/137251807-ba78550a-55db-418d-beb6-e49bfc649af1.png">
For you info I mean tags like this: https://source.datanerd.us/mobile-apps/android_app/tags
Which map to telemetry like this: https://staging.onenr.io/0GbRm2mKWjy

Reply:
@obarney It seems like there's still an open question as to whether browser and mobile are going to make into the release. Is that correct?

Reply:
Yes, it's possible we may have to pull it but we are still trying to resolve the bugs to see if we can fit it in